Sources/PriorityQueue/PriorityQueue+Sequence.swift

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Collections open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension PriorityQueue:Sequence{
+    
+    /*
+     *  Defines an iterator to conform to Sequence
+     */
+    public struct Iterator:IteratorProtocol{
+        //the priority queue at a specific moment
+        private var snapshotPriorityQueue:PriorityQueue
+        
+        public init(snapshotPriorityQueue:PriorityQueue){
```

Comment:
Such initializers do not need to be public. 

```suggestion
        internal init(_base: PriorityQueue) {
```